sematic/plugins/building/docker_builder.py

```diff
@@ -532,26 +533,27 @@ def _generate_dockerfile_contents(
         for data_glob in source_build_config.data:
             data_files = glob.glob(data_glob)
             if len(data_files) > 0:
-                for data_file in data_files:
+                for data_file in sorted(data_files):
                     dockerfile_contents = (
                         f"{dockerfile_contents}\nCOPY {data_file} {data_file}"
                     )
+        # provide an empty line between data and src, for readability
+        dockerfile_contents = f"{dockerfile_contents}\n"
 
     if source_build_config is not None and source_build_config.src is not None:
         logger.debug("Adding source files: %s", source_build_config.src)
         for src_glob in source_build_config.src:
             src_files = glob.glob(src_glob)
             if len(src_files) > 0:
-                for scr_file in src_files:
+                for src_file in sorted(src_files):
```

Comment:
might want a comment that the sort is just for determinism

Reply:
Done.